suggestions/browsers.json

```diff
@@ -5,6 +5,34 @@
         "repo": "https://github.com/GrapheneOS/Vanadium",
         "reason": "GrapheneOS only"
     },
+    {
+        "id": "org.mozilla.firefox",
+        "label": "Firefox",
+        "source": "g",
+        "repo": "https://github.com/mozilla-mobile/firefox-android",
+        "reason": "The browser made by Mozilla, developed with user in mind.\nUses their homemade browser engine: GeckoView, and has a native support for extensions.\nContains telemetry bits that can be disabled in settings"
+    },
+    {
+        "id": "org.mozilla.fennec_fdroid",
+        "label": "Fennec F-Droid",
+        "source": "f",
+        "repo": "https://gitlab.com/relan/fennecbuild",
+        "reason": "The Firefox you know, but with proprietary bits and telemetry removed"
+    },
+    {
+        "id": "com.duckduckgo.mobile.android",
```

Comment:
DDG browser is as bad as their search engine which is basically Bing (and this is basically a WebView).